scalyr_agent/tests/util/json_util_test.py

```diff
@@ -100,6 +104,10 @@ def __runtest(library):
         if sys.version_info[:2] > (2, 5):
             __runtest(JSON)
 
+        # do the same check but now with binary string.
+        if type(text) is six.text_type:
```

Comment:
``isistance`` check is usually preferred, but this also works.

Reply:
Yes, you are right, I should take this as a habit.